jupyter-org-client.el

```diff
@@ -1034,20 +1034,15 @@ parsed, wrap DATA in a minipage environment and return it."
   (with-temp-buffer
     (insert data)
     (goto-char (point-min))
-    ;; Try to determine if we are in an environment or fragment
-    (if (save-excursion
-          (forward-char 2)
-          (org-inside-LaTeX-fragment-p))
-        (org-element-latex-fragment-parser)
-      ;; If we are not in a fragment, try to parse an environment
-      (let ((env (ignore-errors
-                   (org-element-latex-environment-parser
-                    (point-max) nil))))
-        (if (eq (org-element-type env) 'latex-environment) env
-          ;; If all else fails, wrap DATA in a minipage
-          ;; environment
-          (jupyter-org-latex-environment
-           (concat "\
+    (let ((type (org-element-type (org-element-context))))
+      (cond ((eq type 'latex-fragment)
```

Comment:
Since the parser functions essentially return the same thing as `org-element-context` does why don't we return the value of `org-element-context` when `(memq type '(latex-fragment latex-environment))`?

Reply:
Good point, I've updated the PR

Reply:
PS: sorry for the slow response on this